### ISSUE TITLE
Feature request: more consistent file outputs between tools

### ISSUE THREAD

**Issue by thundernixon** [googlefonts/gftools#136]
Currently, I'm working on processing roman + italic variable fonts for Google Fonts.

I need to use four gftools for this: `fix-vf-meta`, `fix-nonhinting`, `fix-gasp`, and `fix-dsig`.

I tend to need these gftools in each font I prep for Google Fonts.

A big point of friction for using these is that these four scripts have different (and somewhat hard to predict) file output names.

```
interUprightVF=$interDir/build/fonts/var/Inter-Roman-VF.ttf
interItalicVF=$interDir/build/fonts/var/Inter-Italic-VF.ttf

# build stat tables for proper style linking

gftools fix-vf-meta $interUprightVF $interUprightVF
gftools fix-vf-meta $interItalicVF $interItalicVF

mv ${interUprightVF/".ttf"/".ttf.fix"} $interUprightVF
mv ${interItalicVF/".ttf"/".ttf.fix"} $interItalicVF

# prevent warnings/issues caused by no hinting tables – this fixes the file in-place

gftools fix-nonhinting $interUprightVF $interUprightVF
gftools fix-nonhinting $interItalicVF $interItalicVF

mv ${interUprightVF/".ttf"/"-backup-fonttools-prep-gasp.ttf"} $interUprightVF
mv ${interItalicVF/".ttf"/"-backup-fonttools-prep-gasp.ttf"} $interItalicVF

# assert google fonts spec for how fonts should rasterize in different contexts

gftools fix-gasp --autofix $interUprightVF
gftools fix-gasp --autofix $interItalicVF

mv ${interUprightVF/".ttf"/".ttf.fix"} $interUprightVF
mv ${interItalicVF/".ttf"/".ttf.fix"} $interItalicVF

# prevent warnings/issues caused by no digital signature tables

gftools fix-dsig --autofix $interUprightVF 
gftools fix-dsig --autofix $interItalicVF
```

## Request

Each gftool could have an argument like `--inplace` / `-i` to edit a file and output it with the original name. This would make pipelines much simpler to set up. Alternatively, every script could allow the `tool <input-font> <output-font>` setup.

Additionally, all edited files could get the same suffix. E.g. every output could be suffixed with `.fix`, to make names such as `inter-regular.ttf.fix`. This would also simplify shell pipelines.

**Comment by alerque** [comment]
YES PLEASE! I came around just to request this. Trying to use gftools in [fontship](https://github.com/theleagueof/fontship) is a real headache because of this. Every subcommand has a different expectation about input/output file names!

1. All tools should be consistent somehow.
2. No tools should _only_ operate in-place. That should be optional for tools that only work on a single file perhaps, but inplace edits are problematic to use in other automated tooling because one cannot verify whether the tools has been run or not.
3. Automatically added suffixes to batch jobs should be configurable. Some people may always want `.fix`, but some workflows will be much easier to support if each tool had a configurable output file name or name format (or at least a configurable output suffix).
